config/prometheus/README.md

```diff
@@ -0,0 +1,15 @@
+# Prometheus Exporter Prerequisites
+
```

Comment:
Add small blurb explaining that prometheus exporter can be scraped by a prometheus server. See the google managed prometheus example.